pkg/forward/forward.go

```diff
@@ -62,18 +62,20 @@ type InterStepDataForward struct {
 	// idleManager manages the idle watermark status.
 	idleManager *wmb.IdleManager
 	// wmbChecker checks if the idle watermark is valid.
-	wmbChecker wmb.WMBChecker
+	wmbChecker           wmb.WMBChecker
+	toVertexPartitionMap map[string]int
```

Comment:
Do we need to have a map to track the partitions? Can we use the `len(val)` of `toBuffers`?

I can imagine it's different for a sink, but does it matter?

Reply:
removed it.